README.md

```diff
@@ -66,34 +66,25 @@ A single build requires about 100 GiB of disk space. The cache directories in
 
 ## Building a BISDN Linux image
 
-### x86_64 platforms
+BISDN Linux supports two different architectures: ARM XGS iProc, and intel x86-64.
```

Comment:
```suggestion
BISDN Linux supports two different architectures: ARM XGS iProc, and Intel x86-64.
```

x86-64 is not an Intel architecture, but we could argue that all supported x86-64 switches have Intel CPUs.

Reply:
Fixed, and added Broadcom to XGS iProc to align it.

> x86-64 is not an Intel architecture, but we could argue that all supported x86-64 switches have Intel CPUs.

Right, we currently only have tested switches with Intel CPUs, and I'm not even sure we even include any drivers for AMD based systems. So I think we should stay explicit with Intel.